uvm/js/common/ungrid/GridController.js

```diff
@@ -1010,8 +1039,60 @@ Ext.define('Ung.cmp.GridController', {
             }
         }
     },
+
+    /** 
+     * The component here is a container which contains nested items i.e. fields inside it.
+     * This function is used to go into all the available nested fields and check if it is required field.
+    */
+    getNestedFieldIfPresent: function(component, me, fieldName){
+        if((component.xtype === 'fieldset' || component.xtype === 'container') && component.items && component.items.length > 0){
+            for(var currItemIdx = 0; currItemIdx < component.items.length; currItemIdx++){
+                if(component.items[currItemIdx].xtype === 'fieldset' || component.items[currItemIdx].xtype === 'container'){
+                    var nestedResult = me.getNestedFieldIfPresent(component.items[currItemIdx], me, fieldName);
+                    if(nestedResult){
+                        return nestedResult;
+                    }
+                } else if (component.items[currItemIdx].bind) {
+                    var reqFieldConfig = me.getFieldNameFromBind(component.items[currItemIdx], fieldName);
+                    if (reqFieldConfig) {
+                        return reqFieldConfig;
+                    }
+                }
+            }
+        }
+    },
+
+    /** 
+     * This function is used to extract the name of the field from current fieldConfig and check
+     * if that is the field which we want
+    */
+    getFieldNameFromBind: function (fieldConfig, fieldName) { 
+        // Extract the field name from bind property in various formats
+        var bindValue = fieldConfig.bind.value || fieldConfig.bind;
```

Comment:
Won't it throw TypeError if fieldConfig.bind is null ?

Reply:
No, it will always have bind value